lib/iris/tests/unit/util/test_new_axis.py

```diff
@@ -143,6 +143,15 @@ def test_lazy_data(self):
         self.assertTrue(res.has_lazy_data())
         self.assertEqual(res.shape, (1,) + cube.shape)
 
+    def test_masked_unit_array(self):
```

Comment:
The indentation should be 4 spaces to be consistent with the rest of the file (and iris as a whole).